OrbitClientModel/CaptureSerializer.cpp

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "OrbitClientModel/CaptureSerializer.h"
+
+#include <fstream>
+#include <memory>
+
+#include "Callstack.h"
+#include "EventTracer.h"
+#include "FunctionUtils.h"
+#include "OrbitBase/MakeUniqueForOverwrite.h"
+#include "OrbitProcess.h"
+#include "absl/strings/str_format.h"
+#include "capture_data.pb.h"
+#include "google/protobuf/io/coded_stream.h"
+#include "google/protobuf/io/zero_copy_stream_impl.h"
+#include "google/protobuf/message.h"
+
+using orbit_client_protos::CallstackEvent;
+using orbit_client_protos::CallstackInfo;
+using orbit_client_protos::CaptureInfo;
+using orbit_client_protos::FunctionInfo;
+using orbit_client_protos::FunctionStats;
+using orbit_client_protos::TimerInfo;
+
+void CaptureSerializer::WriteMessage(const google::protobuf::Message* message,
```

Comment:
The usual way of handling namespaces in cc files is to use them as in the headers:
```
namespace capture_serializer {

void WriteMessage(const google::protobu ...

namespace internal{

CaptureInfo GenerateCaptureInfo( ...

}  // internal

}  // capture_serializer 

```

Reply:
Oh ok, I didn't know; I have updated it; thanks :)